cypress/integration/examples/add_conflicting_product.spec.js

```diff
@@ -0,0 +1,48 @@
+/// <reference types="cypress" />
+
+import Chance from "chance";
+const chance = new Chance();
+
+const randomCheckbox = chance.integer({ min: 0, max: 6 });
+
+describe("Choose new page", () => {
+  it("finds the link to the product page, clicks it and gets directed there", () => {
+    cy.visit("http://localhost:4000");
+    cy.wait(3000);
+    cy.get(".product-link").click();
+    cy.wait(1000);
+  });
```

Comment:
You say that you "... and gets directed there", but this is not tested.
It finds the link and clicks it without checking the redirect.

Something like this is missing:

```js
cy.url().should('include', '/product')
```

Reply:
Thanks for the feedback Leon. I added assertions and made a few other adjustments